src/pages/pool/_id.vue

```diff
@@ -155,10 +155,8 @@ const missingPrices = computed(() => {
 
 const titleTokens = computed<PoolToken[]>(() => {
   if (!pool.value || !pool.value.tokens) return [];
-  const { tokens } = removeBptFrom(pool.value);
```

Comment:
Just to double check if I understood it correctly: 

We don't need `removeBptFrom` because now we will use leafs and bpts are never leafs. Right?

Reply:
Exactly.